images/openshift-golang-builder.Dockerfile

```diff
@@ -3,20 +3,20 @@ FROM ubi7:7-released
 ARG GOPATH
 # This feels a little ugly but it seems to be the simplest way to enable the SCL automatically for docker exec:
 ENV container=oci \
-    MANPATH=/opt/rh/go-toolset-1.18/root/usr/share/man: \
-    X_SCLS=go-toolset-1.18  \
+    MANPATH=/opt/rh/golang-1.18/root/usr/share/man: \
+    X_SCLS=golang-1.18  \
```

Comment:
all this `/opt/rh` stuff is the SCL location. this scratch build isn't packaged as an SCL, so we don't need any of this.
```suggestion
```